build.gradle

```diff
@@ -136,10 +138,23 @@ ext.dropwizardMetricsVersion = '3.1.0'
 ext.findBugsVersion = '3.0.0'
 ext.confluentVersion = '2.0.1'
 
+dependencyManagement {
+  imports {
+    mavenBom 'com.amazonaws:aws-java-sdk-bom:1.11.8'
+  }
+}
+
 ext.externalDependency = [
   "antlrRuntime": "org.antlr:antlr-runtime:3.5.2",
   "avro": "org.apache.avro:avro:" + avroVersion,
   "avroMapredH2": "org.apache.avro:avro-mapred:" + avroVersion + ":hadoop2",
+  "awsCore": "com.amazonaws:aws-java-sdk-core:1.11.8",
```

Comment:
may want to define `ext.awsJavaSdkVersion=1.11.8` so that they can all be upgraded together